site/content/en/docs/Guides/Client SDKs/godot.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Godot Game Server Client SDK"
+linkTitle: "Godot"
+date: 2021-04-15T10:17:50Z
+weight: 15
+description: "This is the Godot version of the Agones Game Server Client SDK."
+---
+
+Check the [Client SDK Documentation]({{< relref "_index.md" >}}) for more details on each of the SDK functions and how to run the SDK locally.
+
+## Download
+
+Download the source {{< ghlink href="sdks/godot" >}}directly from GitHub{{< /ghlink >}}.
+
+## Prerequisites
+
+- Godot >= 3.x
+
+## Usage
```

Comment:
Is there watch support?

https://agones.dev/site/docs/guides/client-sdks/unreal/ says:

> Current the only missing functionality is the WatchGameServer functionality.

Reply:
Good point, so the Swagger file includes the watch path. So, a watch function is generated, but it currently will do the equivalent of curling the endpoint.

Since this does not fully function, do you think it would be best to manually ignore it during generator or..?